apps/cookbook/src/app/component-status/component-status.component.ts

```diff
@@ -432,6 +421,33 @@ export class ComponentStatusComponent implements OnInit, OnDestroy {
       })
     );
   }
+
+  getPageOfIssues(url: string) {
+    const options = {
+      headers: new HttpHeaders({
+        Authorization: 'token ' + environment.oauth.githubToken1 + environment.oauth.githubToken2,
+      }),
+      observe: 'response' as const,
+    };
+    return this.http.get<Observable<any>>(url, options).pipe(
+      map((res) => ({
+        issues: res.body,
+        next: this.next(res),
+      }))
+    );
+  }
+
+  private next(response: HttpResponse<Observable<any>>): string {
```

Comment:
When using the `observe: 'response'` option for `http.get` in `getPageOfIssues()` the returned type will be HttpResponse<T> - note NOT wrapped in an observable.
Also ensure the `http.get` method is typed to return an array (the issues in JSON format):
```suggestion
  private next(response: HttpResponse<Object[]>): string {
```